Woof/Woof/ViewModel/UserRoleManager.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+/// Responsible for managing and actual state of the current user role in the application.
+final class UserRoleManager: ObservableObject {
```

Comment:
Is it worth calling a manager a class that is essentially a view model?

Reply:
Changed the class name to `UserRoleViewModel`